kctf-operator/deploy/operator.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kctf-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: kctf-operator
+  template:
+    metadata:
+      labels:
+        name: kctf-operator
+    spec:
+      serviceAccountName: kctf-operator
+      containers:
+        - name: kctf-operator
+          # Replace this with the built image name
+          image: REPLACE_IMAGE
```

Comment:
Actually I didn't try to run inside the cluster yet because I was trying to code the most part and then test this way but I think this is used when you run it inside the cluster: https://v0-18-x.sdk.operatorframework.io/docs/golang/quickstart/#1-run-as-a-deployment-inside-the-cluster . The operator needs an image to run.

Reply:
Yes, that's right. As mentioned somewhere above we would have to build & push the operator image to some public registry (e.g. gcr.io) and put here the value. Cluster admins don't have to change it.

But we should better do it using some sort of CI/CD, not manually

Reply:
Yes, I will leave a TODO there so we can fix this in a next PR when I start to run inside the cluster